tests/foreman/api/test_activationkey.py

```diff
@@ -6,7 +6,7 @@
 
 :CaseLevel: Acceptance
```

Comment:
I think you might want to change it to `:CaseLevel: Component`. This should fix most of testimony complaints about tier assignment, as well as simply make more sense - things like `test_positive_create_with_description` are testing one component independently of anything else, so they do belong to Component level.